funnel/views/index.py

```diff
@@ -107,14 +99,31 @@ def whoami():
         return jsonify(message="Hmm, so who _are_ you?", code=401)
 
 
-@app.route('/json')
-def all_projects_json():
+@app.route('/past.json')
+@requestargs(('page', int), ('per_page', int))
+def past_projects_json(page=1, per_page=10):
```

Comment:
These should be the same parameter names as in /search (I haven't checked)

Reply:
They are same.